examples/openai_agent_sdk.py

```diff
@@ -0,0 +1,69 @@
+import asyncio
+import os
+
+from agents import Agent, GuardrailFunctionOutput, InputGuardrail, Runner
+from pydantic import BaseModel
+
+import acuvity
+from acuvity import Acuvity, Guard, GuardName
+
+s = Acuvity(
+    security=acuvity.Security(
+       token=os.getenv("ACUVITY_TOKEN", ""),
+    )
+)
+
+class GuardOutput(BaseModel):
+    is_guard_triggered: bool
+    reasoning: str
+
+guardrail_agent = Agent(
+    name="Acuvity Guardrail check",
+    instructions="Check for any exploit in the prompt.",
+    output_type=GuardOutput,
+)
+
+async def prompt_injection_guardrail(ctx, agent, input_data):
+    gc = [Guard.create(GuardName.PROMPT_INJECTION)]
+    matches = s.apex.scan(input_data, guard_config=gc).matches()
+    return GuardrailFunctionOutput(
+        output_info="prompt injection guard triggered",
+        tripwire_triggered=matches[0].response_match == "YES",
+    )
```

Comment:
@abhijitherekar this is a great generic integration example, but I think we should try to offer this in an even more compact form:

- imagine we make `agents` an optional dependency for us
- we can then have a conditional import for agents in our own library code
- and have a function on `s.apex` which can directly return a guardrail function or maybe even better an `InputGuardrail` instance based on guards that we provide

so something along the lines of:

```
Agent(
    ...
    input_guardrails=[s.apex.AgentInputGuardrail(GuardName.PROMPT_INJECTION)]
)
```

Reply:
Yes sure, I can take it up late. 
Should we merge this for now. @spgenaic wanted to show this functionality to few people. 

He wants to share the public link of the example.

Reply:
sure, please create at least an issue then though for the real integration work